bundles/org.openhab.ui/web/src/components/item/item-form.vue

```diff
@@ -23,11 +23,11 @@
         </select>
       </f7-list-item>
       <!-- Use v-show instead of v-if, because otherwise the autocomplete for category would take over the unit -->
-      <f7-list-input v-show="!hideType && item.type && item.type.startsWith('Number:') && createMode" label="Unit" type="text" :value="item.unit"
-                     info="Used internally, for persistence and external systems. It is independent from the state visualization in the UI, which is defined through the state description."
+      <f7-list-input v-show="!hideType && item.type && item.type.startsWith('Number:')" ref="unit" label="Unit" autocomplete="off" type="text" :value="item.unit"
+                     :info="(createMode) ? 'Type any valid unit or select from one of the proposed (non-exhaustive list of) units. Used internally, for persistence and external systems. It is independent from the state visualization in the UI, which is defined through the state description.' : ''"
```

Comment:
Any reason why this applies only to createMode and not to editMode?

Reply:
We are still looking for a way to have these info messages hidden behind a question mar link when not in create mode, see also @jimtng PR. It makes the screen less cluttered and is mostly relevant when creating.